extensions/hibernate-reactive/deployment/src/main/java/io/quarkus/hibernate/reactive/deployment/HibernateReactiveProcessor.java

```diff
@@ -85,12 +85,13 @@ void registerBeans(BuildProducer<AdditionalBeanBuildItem> additionalBeans, Combi
             List<PersistenceUnitDescriptorBuildItem> descriptors,
             JpaModelBuildItem jpaModel) {
         if (!hasEntities(jpaModel)) {
-            LOGGER.warnf("Skipping registration of %s and %s because no JPA entities were found",
-                    ReactiveSessionFactoryProducer.class.getSimpleName(), ReactiveSessionProducer.class.getSimpleName());
             return;
         }
 
         if (descriptors.size() == 1) {
+            LOG.warnf(
+                    "Skipping registration of %s and %s because exactly one persistence unit is required for their registration",
+                    ReactiveSessionFactoryProducer.class.getSimpleName(), ReactiveSessionProducer.class.getSimpleName());
```

Comment:
This should be in a new `else` after this `if`. If we get here, the beans *will* be registered. We want to display the warning in the *other* case.

Also, you should remove the `if (!hasEntities(jpaModel)) { return }` above, otherwise your warning won't always be displayed.

Reply:
I moved the warning to an `else` block. Sorry about that, that was my mistake, not a problem with your comments. 😄

The `hasEntities` check is gone too.